src/main/java/net/minecraftforge/fml/common/FMLCommonHandler.java

```diff
@@ -202,7 +202,7 @@ public Side getSide()
     public Side getEffectiveSide()
     {
         Thread thr = Thread.currentThread();
-        if (thr.getName().equals("Server thread") || thr.getName().startsWith("Netty Server IO"))
+        if (thr.getName().equals("Server thread") || thr.getName().startsWith("Netty Epoll Server IO") || thr.getName().startsWith("Netty Server IO"))
```

Comment:
My point was, as it's not going to work properly in all cases regardless, why not keep it simple? Is misidentifing as client particularly preferable to misidentifing as server?